test/verify/check-services

```diff
@@ -267,6 +267,16 @@ Unit=test.service
         self.toggle_onoff()
         self.check_service_details(["Running", "Automatically starts"], ["Reload", "Restart", "Stop", "Disallow running (mask)"], True)
 
+        # Survives a burst of events
+        b.go("#/")
+        self.wait_service_present("test.service")
+        m.execute("udevadm trigger; udevadm settle")
+        with b.wait_timeout(20):
```

Comment:
This seems rather brittle, relying on a "fast enough" response. We know that's going to bite us. 

That's what I meant with "we are entering benchmarking" territory.  What we really want is find performance regressions, so we should measure how long something takes, in a carefully controlled environment or in a way that doesn't get fooled by noise from the environment.  E.g. "number of calls to `render`".

Reply:
I'm happy to drop the wait_timeout, it fails without that even. I just started with that, and then realized that we don't really need it.